WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -5490,7 +5490,7 @@
 	name="Lamech"
 	# Lamech ibn Napyot Ghifarid
 	dynasty=10424446
-	religion="jewish" #Banu Ghifar was a jewish tribe
+	religion="semitic_pagan"
 	culture="adnanite"
```

Comment:
I think you accidentally reverted plenty of entries.

Reply:
Oh, It seems like it. How do I solve it? Also, I still have to do some merging that I will do later today.

Reply:
Revert to master branch version of the file and then re-add you actual changes on top.

Reply:
Solved